tools/hack/create-cluster.sh

```diff
@@ -6,12 +6,32 @@ set -euo pipefail
 CLUSTER_NAME=${CLUSTER_NAME:-"envoy-gateway"}
 METALLB_VERSION=${METALLB_VERSION:-"v0.13.10"}
 KIND_NODE_TAG=${KIND_NODE_TAG:-"v1.28.0"}
+NUM_WORKERS=${NUM_WORKERS:-""}
+
+
+KIND_CFG=$(cat <<-EOM
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+EOM
+)
+
```

Comment:
nit: can you add https://kind.sigs.k8s.io/docs/user/quick-start/#multi-node-clusters this in here ?